docs/manuals/core/entities/task-mgt.md

```diff
@@ -1,4 +1,4 @@
-In this section, it is assumed that [`my_config.py`](../my_config.py) module contains a Taipy configuration
+In this section, it is assumed that [`my_config.py`](https://github.com/Avaiga/taipy-doc/blob/develop/docs/manuals/core/my_config.py) module contains a Taipy configuration
```

Comment:
We cannot have an absolute link for that. Here you are pointing to the develop version of the file which might change over time and over the versions.